components/index.tsx

```diff
@@ -0,0 +1,7 @@
+import { Header } from './Header';
+import { Layout } from './Layout';
+
+export {
+  Header,
+  Layout,
+};
```

Comment:
Do you use both the JavaScript and TypeScript versions of the index file? Also, you have two 'index.tsx' files. Why is that?

Reply:
The `component/index.tsx` could be a js file instead but I do not know if in the future I would need Typescript in it since it make scale up to be a very large file.

Both of these `index.tsx` files serve different purposes since they are in different locations. Review them and you may understand.

Reply:
Are you doing this so you don't have to go into folders when importing components? For example, doing `@/components` instead of `@/components/Header/header.tsx`?

Reply:
Correct!